packages/pinia/src/types.ts

```diff
@@ -572,7 +572,11 @@ export type _ActionsTree = Record<string, _Method>
  * For internal use **only**
  */
 export type _ExtractStateFromSetupStore_Keys<SS> = keyof {
```

Comment:
This type shouldn't include computed (even writable ones). The types related to `mapWritableState` should be the ones changed so they include writable computed alongside state properties

Reply:
Ok, I get it